conda/dev.yaml

```diff
@@ -15,3 +15,4 @@ dependencies:
   - sqlalchemy
   - pip:
     - pysus
+    - sodapy
```

Comment:
it is missing an empty line at the end of the file

Reply:
Do you know why it returned an poetry error? Does conda pip uses poetry?

Can't install soday through poetry even tho poetry finds the right version?
```
Using version ^2.2.0 for sodapy


  NonExistentKey

  'Key "sodapy" does not exist.'
```

Reply:
not sure ... can you share the entire log pls?

Reply:
```
11:04 $ poetry add sodapy
Using version ^2.2.0 for sodapy


  NonExistentKey

  'Key "sodapy" does not exist.'

  at ~/miniconda3/envs/epigraphhub/lib/python3.8/site-packages/tomlkit/container.py:650 in __getitem__
      646│             key = SingleKey(key)
      647│ 
      648│         idx = self._map.get(key, None)
      649│         if idx is None:
    → 650│             raise NonExistentKey(key)
      651│ 
      652│         if isinstance(idx, tuple):
      653│             # The item we are getting is an out of order table
      654│             # so we need a proxy to retrieve the proper objects
```

Reply:
It works via pip thats why I added it to conda pip deps

Reply:
we should keep the conda dependencies as short as possible ... just for base dependencies ...
the directly dependencies should always be located at the pyproject.toml file